Describe further parameters of the cds-services-archetype. (#1589)
cap-js/docs: java/developing-applications/building.md

| Option | Description |
| -- | -- |
| `-DgroupId=<a valid Maven groupId>` | The `groupId` of the Maven artifact for the new project. If not specified, Maven prompts for user input. |
| `-DartifactId=<a valid Maven artifactId>` | The `artifactId` of the Maven artifact for the new project. If not specified, Maven prompts for user input. |

> | `-Dversion=<a valid Maven version>` | The `version` of the new Maven artifact for the new project. Defaults to `1.0.0-SNAPSHOT` |

Reviewer: ```suggestion
| `-Dversion=<a valid Maven version>` | The `version` of the Maven artifact for the new project. Defaults to `1.0.0-SNAPSHOT` |
```

> | `-Dpackage=<a valid Java package>` | Specifies the Java package of the originally created classes. Defaults to `${groupId}.${artifactId}`. |

Reviewer: ```suggestion
| `-Dpackage=<a valid Java package>` | The Java package for your project's classes. Defaults to `${groupId}.${artifactId}`. |
```
